### ISSUE TITLE
pretty-quick --staged should give better indication of which file failed 

### ISSUE THREAD

**Issue by jcollum-nike** [azz/pretty-quick#45]
```./node_modules/.bin/pretty-quick --staged
🔍  Finding changed files since git revision 1973849.
🎯  Found 26 changed files.
/Users/jcol53/work/orders/node_modules/prettier/index.js:24285
      throw error;
      ^

SyntaxError: Unexpected token (1:1)
> 1 | 
    | ^
    at createError (/Users/jcol53/work/orders/node_modules/prettier/parser-babylon.js:1:110)
    at Object.parse (/Users/jcol53/work/orders/node_modules/prettier/parser-babylon.js:1:986)
   at Object.parse$4 [as parse] (/Users/jcol53/work/orders/node_modules/prettier/index.js:24274:19)
    at formatWithCursor (/Users/jcol53/work/orders/node_modules/prettier/index.js:33650:27)
    at format (/Users/jcol53/work/orders/node_modules/prettier/index.js:33695:10)
    at format (/Users/jcol53/work/orders/node_modules/prettier/index.js:33945:12)
    at exports.default (/Users/jcol53/work/orders/node_modules/pretty-quick/dist/formatFiles.js:18:41)
    at exports.default (/Users/jcol53/work/orders/node_modules/pretty-quick/dist/index.js:47:29)
    at Object.<anonymous> (/Users/jcol53/work/orders/node_modules/pretty-quick/bin/pretty-quick.js:12:1)
```

Seems clear that there is a syntax error but I have 25 files staged (big merge). It would be helpful if there was a `--verbose` flag that would output the name of each file as they were processed, that way I can quickly figure out which file is causing the issues.

**Comment by jcollum-nike** [comment]
Workaround: 

1. `git diff --name-only --cached`
1. run `./node_modules/.bin/prettier` on every file from item 1 in a bash script

`xargs` or join each line with `&&` might help too

**Comment by azz** [comment]
Definitely a good idea.

**Comment by flurmbo** [comment]
Can I try this one?

**Comment by azz** [comment]
Released in 1.8.0
